functional-tests/utils.py

```diff
@@ -49,9 +49,9 @@ def generate_n_blocks(bitcoin_rpc: BitcoindClient, n: int):
     print(f"generating {n} blocks to address", addr)
     try:
         blk = bitcoin_rpc.proxy.generatetoaddress(n, addr)
-        print("made blocks", blk)
+        print(f"made blocks {blk}")
     except Exception as ex:
-        log.warning(f"{ex} while generating address")
+        logging.warning(f"{ex} while generating address")
```

Comment:
Does this direct call via the module end up getting captured by the test-level logger?

Reply:
Since this is root level logger, i don't think it's captured by test-level logger.